R/breaks.r

```diff
@@ -150,3 +150,42 @@ cbreaks <- function(range, breaks = extended_breaks(), labels = scientific_forma
 
   list(breaks = breaks, labels = labels)
 }
+
+#' Minor breaks.
```

Comment:
Titles shouldn't end in a full stop (I've inconsistently applied this rule in the past, but I've been trying to do better going forward)